AirCasting/Graph/Graph.swift

```diff
@@ -79,22 +79,25 @@ struct Graph: UIViewRepresentable {
     }
     
     func getMidnightsPoints(startingDate: Date) -> [Double] {
-        let now = Date()
+        let day = Date()
+        // [SMALL HACK] - By adding a day to the current Date we are ensuring
+        // that session which is currently recording will be able to show midnight line
+        let now = Calendar.current.date(byAdding: .day, value: 1, to: day)!
```

Comment:
I think this shouldn't be named "now" - if I understand it correctly it's more like `nextDay`?

Reply:
yeah, definitely I will change it